.github/workflows/issue-pr-triage.yml

```diff
@@ -3,33 +3,48 @@ name: Auto-triage to project boards
 on:
   issues:
     types: [opened, labeled]
-  pull_request:
+  pull_request_target:
     types: [opened, labeled]
 
 env:
-  GITHUB_TOKEN: ${{ secrets.OPENSOURCE_BOT_TOKEN }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  COLUMN_NAME: Needs Triage # column where issues will be created
+  REPOSITORY: ${{ github.repository }}
 
 jobs:
   content_project:
     runs-on: ubuntu-latest
+    env:
+      PROJECT_ID: 11873277
+      PROJECT_NAME: Docs PRs and Issues
     name: Triage to Content Project
     steps:
     - name: Triages NEW issues and NEW pull requests to the Content Project
-      uses: srggrs/assign-one-project-github-action@1.2.0
       if: github.event.action == 'opened'
-      with:
-        project: 'https://github.com/newrelic/docs-website/projects/3'
-        column_name: 'Needs Triage'
+      run: |
+        PR_NUMBER=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
```

Comment:
I think this could be simplified to just:

```sh
PR_NUMBER=${{ github.event.issue.number }}
```

[Relevant docs](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#issue_comment)

Reply:
We could even remove it and use it inline for further simplification

Reply:
change made